From 554d7021e0f413186e0124a264fcb3c59ef92bad Mon Sep 17 00:00:00 2001
From: Damien Elmes <git@ichi2.net>
Date: Wed, 21 Apr 2010 13:31:59 +0900
Subject: [PATCH] don't specify an input index by default

---
 anki/sound.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/anki/sound.py b/anki/sound.py
index 9943a83..2272bd2 100644
--- a/anki/sound.py
+++ b/anki/sound.py
@@ -238,7 +238,7 @@ try:
     PYAU_FORMAT = pyaudio.paInt16
     PYAU_CHANNELS = 1
     PYAU_RATE = 44100
-    PYAU_INPUT_INDEX = 0
+    PYAU_INPUT_INDEX = None
 except:
     pass
 
-- 
1.6.2.5

