diff -uNr anki-0.9.9.6.old/ankiqt/config.py anki-0.9.9.6/ankiqt/config.py
--- anki-0.9.9.6.old/ankiqt/config.py	2009-01-21 14:53:21.000000000 +0100
+++ anki-0.9.9.6/ankiqt/config.py	2009-02-11 23:18:16.000000000 +0100
@@ -44,7 +44,7 @@
             'iconSize': 32,
             'syncOnLoad': False,
             'syncOnClose': False,
-            'checkForUpdates': True,
+            'checkForUpdates': False,
             'interfaceLang': "",
             'syncUsername': "",
             'syncPassword': "",
