diff -uNr anki-0.9.9.7.8.old/ankiqt/config.py anki-0.9.9.7.8/ankiqt/config.py
--- anki-0.9.9.7.8.old/ankiqt/config.py	2009-05-06 08:35:07.000000000 +0200
+++ anki-0.9.9.7.8/ankiqt/config.py	2009-05-06 21:27:39.000000000 +0200
@@ -45,7 +45,7 @@
             'syncOnLoad': True,
             'syncOnClose': True,
             'syncInMsgBox': False,
-            'checkForUpdates': True,
+            'checkForUpdates': False,
             'interfaceLang': "",
             'syncUsername': "",
             'syncPassword': "",
