diff -uNr anki-0.9.9.6.old/libanki/setup.py anki-0.9.9.6/libanki/setup.py
--- anki-0.9.9.6.old/libanki/setup.py	2009-01-21 14:51:33.000000000 +0100
+++ anki-0.9.9.6/libanki/setup.py	2009-02-13 22:14:15.000000000 +0100
@@ -25,8 +25,8 @@
       url='http://ichi2.net/anki/index.html',
       license='GPLv3',
       packages=["anki", "anki.features", "anki.features.chinese", "anki.importing"],
-      package_data={'anki':
-                    ['samples/*','locale/*/*/*']},
+      package_data={'anki': ['samples/*','locale/*/*/*'],
+		    'anki.features.chinese': ['unihan.db']},
       include_package_data=True,
       zip_safe=False,
       install_requires=[
