diff -uNr anki-0.9.9.7.1.old/ankiqt/config.py anki-0.9.9.7.1/ankiqt/config.py
--- anki-0.9.9.7.1.old/ankiqt/config.py	2009-03-23 05:40:26.000000000 +0100
+++ anki-0.9.9.7.1/ankiqt/config.py	2009-03-29 23:24:00.000000000 +0200
@@ -44,7 +44,7 @@
             'iconSize': 32,
             'syncOnLoad': True,
             'syncOnClose': True,
-            'checkForUpdates': True,
+            'checkForUpdates': False,
             'interfaceLang': "",
             'syncUsername': "",
             'syncPassword': "",
