diff --git a/anki/deck.py b/anki/deck.py
index 36982b8..e013b13 100644
--- a/anki/deck.py
+++ b/anki/deck.py
@@ -3779,8 +3779,11 @@ update cards set type = type - 3 where type between 0 and 2 and priority = -3"""
             path = "sqlite:///" + path
         if pool:
             # open and lock connection for single use
+            from sqlalchemy.pool import SingletonThreadPool
+            # temporary tables are effectively useless with the default
+            # settings in 0.7, so we need to force the pool class
             engine = create_engine(path, connect_args={'timeout': 0},
-                                   strategy="threadlocal")
+                                   poolclass=SingletonThreadPool)
         else:
             # no pool & concurrent access w/ timeout
             engine = create_engine(path,
diff --git a/anki/db.py b/anki/db.py
index e9aa207..3ca6216 100644
--- a/anki/db.py
+++ b/anki/db.py
@@ -30,7 +30,7 @@ from sqlalchemy import create_engine
 from sqlalchemy.orm import mapper, sessionmaker as _sessionmaker, relation, backref, \
      object_session as _object_session, class_mapper
 from sqlalchemy.sql import select, text, and_
-from sqlalchemy.exceptions import DBAPIError, OperationalError
+from sqlalchemy.exc import DBAPIError, OperationalError
 from sqlalchemy.pool import NullPool
 import sqlalchemy
 
