diff -uNr anki-1.0.old/ankiqt/config.py anki-1.0/ankiqt/config.py
--- anki-1.0.old/ankiqt/config.py	2010-07-21 05:55:21.000000000 +0200
+++ anki-1.0/ankiqt/config.py	2010-08-02 21:09:29.000000000 +0200
@@ -43,7 +43,7 @@
         fields = {
             'addZeroSpace': False,
             'alternativeTheme': False,
-            'checkForUpdates': True,
+            'checkForUpdates': False,
             'colourTimes': True,
             'created': time.time(),
             'deckBrowserNameLength': 30,
